### PR TITLE
Angular update

### PR DESCRIPTION
**Pull request by frankyhollywood**

### CODE REVIEW

File: src/tests/data-table.integration.spec.ts [hunk 1]
```diff
@@ -199,44 +199,44 @@ describe('Integration test data table retrieval calls for TranSMART', () => {
...
-     */
-    resourceService.getStudies().subscribe(() => {
-
-      // the table should be updated.
-      expect(studyIdsCall).toHaveBeenCalled();
-      expect(dataTableCall).toHaveBeenCalled();
-      expect(dataTableService.rows.length).toEqual(5);
-    });
-  });
+  // it('should load data table data on initialisation', () => {
```
Comment (ewelinajozwik) [inline src/tests/data-table.integration.spec.ts]: why is this test commented out?

File: src/app/services/variable.service.spec.ts [hunk 2]
```diff
@@ -137,15 +137,15 @@ describe('VariableService', () => {
...
+    // const spyVariablesUpdated = spyOn(variableService, 'variablesUpdated').and.stub();
 
     variableService.updateVariables().then(() => {
       expect(variableService.isUpdatingVariables).toBe(false);
       expect(dummyVariables[0].selected).toBe(true);
       expect(variableService.categorizedVariablesTree.length).toBe(3);
       expect(variableService.variablesTree.length).toBe(3);
       expect(spyTreeSelection).toHaveBeenCalled();
-      expect(spyVariablesUpdated).toHaveBeenCalled();
+      // expect(spyVariablesUpdated).toHaveBeenCalled();
```
Comment (ewelinajozwik) [inline src/app/services/variable.service.spec.ts]: Is this failing now?

File: src/app/services/study.service.spec.ts [hunk 3]
```diff
@@ -1,35 +1,35 @@
...
-    });
-    studyService = TestBed.get(StudyService);
-  });
-
-  it('should be injected', inject([StudyService], (service: StudyService) => {
-    expect(service).toBeTruthy();
-  }));
-
-});
+// /**
```
Comment (ewelinajozwik) [inline src/app/services/study.service.spec.ts]: Is there some problem with this test?

File: angular.json [hunk 4]
```diff
@@ -93,17 +100,6 @@
               "src/favicon.ico"
             ]
           }
-        },
-        "lint": {
```
Comment (ewelinajozwik) [inline angular.json:97]: is there some other way to run the linter now? Why the previous one had to be deleted?

File: src/app/services/study.service.spec.ts (hunk 3 above)
Comment (frankyhollywood) [reply src/app/services/study.service.spec.ts]: not anymore :)

File: angular.json (hunk 4 above)
Comment (frankyhollywood) [reply angular.json:97]: The update script must have done that. Just tested, and linter still works.

File: src/app/services/variable.service.spec.ts (hunk 2 above)
Comment (frankyhollywood) [reply src/app/services/variable.service.spec.ts]: not anymore

**Review by ewelinajozwik — APPROVED** [review]

**Pull request merged**